### PR TITLE
Feature/update state hooks

### PR DESCRIPTION
**Pull request by annamkummer**
What (if any) features are you implementing?
- Component Architecture, initial gameboard layout
What (if anything) did you refactor?
- Hooks
Were there any issues that arose?
- n/a
Is there anything that you need from your teammate?
- n/a
Any other comments, questions, or concerns?
-

### CODE REVIEW

File: src/components/Game.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+//   }
+
+function Game({ region }) {
+  // let region = useParams().region
+
+  // useEffect(() => {
+  //   setRegion(region);
+  // }, [region]);
+  
+  return (
```
Comment (jkiernan12) [inline src/components/Game.js:33]: Nice job fleshing out the game page content!

File: src/components/Region.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { useParams } from 'react-router-dom'
+import Nav from './Nav'
+import Game from './Game'
+
+function Region() {
```
Comment (jkiernan12) [inline src/components/Region.js:5]: Good idea shifting useParams here 👍

**Review by jkiernan12 — APPROVED** [review]
Looks great! Thanks for all your work on this!!!

**Review by NEwertKrocker — APPROVED** [review]
This all looks great. A flexible architecture that gives us the foundation we need to build the rest of our app.

**Pull request merged**